addons/website_sale/static/src/js/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -2,10 +2,205 @@
 
 import { WysiwygAdapterComponent } from '@website/components/wysiwyg_adapter/wysiwyg_adapter';
 import { patch } from "@web/core/utils/patch";
+import { Markup } from "@web/legacy/js/core/utils";
 
 // TODO this whole patch actually seems unnecessary. The bug it solved seems
 // to stay solved if this is removed. To investigate.
```

Comment:
This comment should move to the same location as the piece of the patch that it corresponds to.